src/http/responses/dfract.response.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AssetInfo {
```

Comment:
File is incorrectly named, and should be `asset-info.response.ts` AND `asset-historical.response.ts`